tests/conftest.py

```diff
@@ -39,6 +39,14 @@ def pytest_addoption(parser):
         help="Benchmark only option. Only applies to bigint testing."
              "Maximum number of bits, so values > 2**max_bits will wraparound. -1 is interpreted as no maximum."
     )
+    parser.addoption(
+        "--alpha", action="store", default="1",
```

Comment:
in the original alpha is a real value, i just wanna make sure we'll still be multiplying by `1.0` here

Reply:
Added the `1.0` multiplication to the non-bigint cases - verified with @pierce314159 that bigint can't accept a `real` multiplication so it has to remain `int` in that case